static/js/whyis_vue/components/table-view/frontend/src/main.js

```diff
@@ -0,0 +1,45 @@
+// The Vue build version to load with the `import` command
+// (runtime-only or standalone) has been set in webpack.base.conf with an alias.
+import Vue from 'vue';
+import App from './App';
+import router from './router';
+import MockApi from './api';
+/* eslint-disable */
+Vue.config.productionTip = false;
+
+const Tabulator = require('tabulator-tables');
+
+/* eslint-disable no-new */
+new Vue({
+  el: '#app',
+  router,
+  components: { App },
+  template: '<App/>',
+  methods: {
+    testApiPost() {
```

Comment:
Pass tabulator down as a prop, don't have the child reach up.
https://vuejs.org/v2/guide/components.html#Passing-Data-to-Child-Components-with-Props

Reply:
Used events instead of props, so the child components don't receive anything